### PR TITLE
Fix for Issue #59 

### LINKED ISSUE #59: Scrolling preview image bug in mobile view
**Describe the bug**
Scrolling the preview image in mobile view also scrolls the the Heading Text ("Input"), Sub heading ("Type/Paste text here") and the Button ("Draw(Beta)").

**To Reproduce**
1. Switch to mobile view
2. Right swipe on the preview image
3. You'll see that the headings and buttons are now not visible.

**Expected behavior**
The headings and button should remain despite you chosing to scroll through the image.
Error1
Error2

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chrome
 - Version: Version 81.0.4044.129 (Official Build) (64-bit)

**Additional context**
Although a minor bug but it'd be great if the behaviour was as expected.

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Fixes #59. I've made a new div as a container for the three divs: a div containg heading and subheadinig, a div containing image preview, a div containing the button. I've given two classes to this div: flex-1 (exsisting class) and page-container-super (created class). The created class uses `display:flex; flex-direction:column;` so that the three divs align like they did previously. This fixes the issue without breaking any of the exsisting code.

### CODE REVIEW

File: .vscode/snipsnap.code-snippets [hunk 1]
```diff
@@ -0,0 +1 @@
...ody":["cy.server();"],"description":"Start a server to begin routing responses to cy.route() and to change the behavior of network requests.","scope":"javascript,typescript"},"cypress-viewport-2fe6d86":{"prefix":["cypress viewport"],"body":["cy.viewport(${1:width}, ${2:height});"],"description":"Control the size of the screen for your application.","scope":"javascript,typescript"},"cypress-viewport-preset-d816140":{"prefix":["cypress viewport preset"],"body":["cy.viewport('${1:preset}');"],"description":"Control the size of the screen for your application via presets. Preset supports the following options: ipad-2, ipad-mini, iphone-3, iphone-4, iphone-5, iphone-6, iphone-6+, iphone-x, iphone-xr, macbook-11, macbook-13, macbook-15, samsung-note9, samsung-s10","scope":"javascript,typescript"},"cypress-visit-3176543":{"prefix":["cypress visit"],"body":["cy.visit('${1:url}');"],"description":"Visit a remote URL","scope":"javascript,typescript"},"cypress-wait-859e0d0":{"prefix":["cypress wait"],"body":["cy.wait(${1:time});"],"description":"Wait for a number of milliseconds or wait for an aliased resource to resolve before moving on to the next command.","scope":"javascript,typescript"}}
```
Comment (saurabhdaware) [inline .vscode/snipsnap.code-snippets]: this

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Thank you for reporting and fixing bug and everything looks great! 

I guess your vscode configs came in the PR, would you mind deleting it? I should've added this in .gitignore actually 🤦‍♂️

**Comment by RinkiyaKeDad** [comment]
Hey @saurabhdaware, I've removed my vscode configs. It should be good to go now.

**Review by saurabhdaware — APPROVED** [review]
Thank you @RinkiyaKeDad 🎉 

P.S. Nice username xD

**Comment by ForceGT** [comment]
It's really cool to see what a superb community this tool has made

**Comment by saurabhdaware** [comment]
> It's really cool to see what a superb community this tool has made

Rightttt!! It grew up so well 🎉 Thank youu :unicorn:

**Pull request merged**